cli/schedule.go

```diff
@@ -72,25 +72,67 @@ func (r *RootCmd) schedules() *clibase.Cmd {
 	return scheduleCmd
 }
 
+// scheduleShow() is just a wrapper for list() with some different defaults.
 func (r *RootCmd) scheduleShow() *clibase.Cmd {
+	var (
+		filter    cliui.WorkspaceFilter
+		formatter = cliui.NewOutputFormatter(
+			cliui.TableFormat(
+				[]scheduleListRow{},
+				[]string{
+					"workspace",
+					"starts at",
+					"starts next",
+					"stops after",
+					"stops next",
+				},
+			),
+			cliui.JSONFormat(),
+		)
+	)
 	client := new(codersdk.Client)
 	showCmd := &clibase.Cmd{
-		Use:   "show <workspace-name>",
-		Short: "Show workspace schedule",
+		Use:   "show [<workspace>] | [--search <query>] [--all]",
```

Comment:
I'm not sure of the correct way to visualize the requirement, maybe something like this?

```suggestion
		Use:   "show <workspace | --search <query> | --all>",
```

Since `<` is required, and `[` is optional, I feel this best represents the requirements for the command.